test/integration/test-actions.js

```diff
@@ -17,7 +17,7 @@
 import {AmpEvents} from '../../src/amp-events';
 import {createFixtureIframe, poll} from '../../testing/iframe.js';
 
-describe('on="..."', () => {
+describe.configure().run('on="..."', () => {
```

Comment:
Per my understanding, changing `describe(` to `describe.configure().run(` is a no-op, since there's nothing between the `configure()` and the `run(`. Also, since IE tests are opt-in, a plain `describe(` block will not run on IE.

Is this different from what you're seeing?

Reply:
`describe` by itself appears to just be the Mocha global. `__init_tests.js` is what adds `describe.configure`, which actually instantiates the custom `TestConfig` we use, and it's within that constructor that IE is skipped by default. That's why adding it here causes the test to skip:
Here:
![image](https://user-images.githubusercontent.com/6694512/94302702-9fa81400-ff3a-11ea-8134-8e4598f8454c.png)
Elsewhere:
![image](https://user-images.githubusercontent.com/6694512/94302725-acc50300-ff3a-11ea-8590-845faffc08a3.png)

Reply:
Note that above, you can see A) 21 tests are executed (not 22, which included the one that failed) B) the failure caused all other tests in that file not to register, whereas now it counts them as skipped (ie. 466 vs 469)

Reply:
Interesting. Thanks for the clarification. Another reason in favor of removing legacy `describe` tests (#24144).